mypy check fails with Alembic 1.7.3

This issue was discovered by the ci run of #1182.
Mypy check fails with the following error:

> anitya/db/migrations/env.py:21: error: Argument 1 to "fileConfig" has incompatible type "Optional[str]"; expected "Union[Union[str, bytes, PathLike[str], PathLike[bytes]], IO[str], RawConfigParser]"

Forcing Alembic version to be 1.7.1 in `requirements.txt` fixes the issue.

--- a/anitya/db/migrations/env.py
+++ b/anitya/db/migrations/env.py
@@ -18,7 +18,7 @@ config = context.config  # type: ignore
 
 # Interpret the config file for Python logging.
 # This line sets up loggers basically.
-fileConfig(config.config_file_name)
+fileConfig(str(config.config_file_name))
 
 # add your model's MetaData object here
 # for 'autogenerate' support

